VogtPayroll2/Program.cs

```diff
@@ -1,12 +1,102 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace VogtPayroll2
 {
     class Program
     {
+        private static int employeeCounter;
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            PayrollManager payrollManager = new PayrollManager();
+            List<Employee> employeeList = new List<Employee>();
+
+            employeeList = PrintMenu();
+
+            foreach (var employee in employeeList)
+            {
+                employee.DisplayEmployeeInfo(employee);
+            }
+
+
+            
+        }
+
+        public static Employee[] EmployeeArray(int result)
+        {
+            Employee[] employeeArray = new Employee[result];
+
+            return employeeArray;
+        }
+
+        public static int HowManyEmployees()
+        {
+            bool run;
+
+
+            Console.WriteLine("How many employees would you like to add? Please select 1-10");
+            run = int.TryParse(Console.ReadLine(), out int result) && result > 1 && result < 10;
+
+            Console.WriteLine($"Adding {result} employees!");
+
+            return result;
+
+        }
+
+        public static List<Employee> PrintMenu()
+        {
+            PayrollManager payrollManager = new PayrollManager();
+            List<Employee> employeeList = new List<Employee>();
+            char option;
+            string name;
+            int hoursWorked;
+            decimal salary;
+            decimal payrate;
+
+
+            Console.WriteLine("Press 'q' to quit, 's' to create a salary employee and 'h' for an hourly employee");
+            option = Console.ReadKey().KeyChar;
+            Console.WriteLine("");
+
+            while (option != 'q')
+            {
+   
+                
+                switch (option)
+                {
+                    case 's':
+                        Console.WriteLine("What is the name of the employee?");
+                        name = Console.ReadLine();
+                        Console.WriteLine("How many hours were worked?");
+                        hoursWorked = Convert.ToInt32(Console.ReadLine());
+                        Console.WriteLine("What is the salary of the employee?");
+                        salary = Convert.ToDecimal(Console.ReadLine());
+                        employeeList = payrollManager.AddEmployee(PayrollConsoleReader.ReadSalaryEmployeeFromConsole(name, hoursWorked, salary));
+                        employeeCounter++;
```

Comment:
Move this logic to the PayrollConsoleReader. The purpose of the PayrollConsoleReader is to separate out the responsibility for knowing how to parse the employee. This Program.cs class doesn't need to know how -- it just needs to trust that the PayrollConsoleReader knows how.

In the end, this should look like:
```c#
payrollManager.AddEmployee(PayrollConsoleReader.ReadSalaryEmployeeFromConsole());
```

Notice that you don't need any params -- Program.cs never needs to know how an employee is even built.
```